<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RAMANAIDU" failures="1" tests="3" name="Test" time="10.019" errors="0" timestamp="2025-06-14T10:59:50 IST">
  <testcase classname="testng.OrangeHRMTest" name="testLogo" time="0.097"/>
  <testcase classname="testng.OrangeHRMTest" name="testAppUrl" time="0.036">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at testng.OrangeHRMTest.testAppUrl(OrangeHRMTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="testng.OrangeHRMTest" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
